user-interface/src/search/SearchScreen.tsx

```diff
@@ -167,9 +199,12 @@ export default function SearchScreen() {
                   className="new-court__select"
                   label="District (Division)"
                   ariaLabelPrefix="District (Division)"
+                  ariaDescription="Select multiple Districts (Divisions)."
```

Comment:
Are we sure on this wording? "Select multiple" could be perceived as requiring more than one selection. I would think we would want to indicate that zero, one, or more selections are valid. Or maybe just say something like "multi-select [field]"?

Reply:
@shalliedragon